app/src/main/java/com/invisibleteam/goinvisible/mvvm/settings/SettingsViewModel.java

```diff
@@ -0,0 +1,17 @@
+package com.invisibleteam.goinvisible.mvvm.settings;
+
+
+import android.databinding.ObservableBoolean;
+
+public class SettingsViewModel {
+
+    private final ObservableBoolean isEnabled = new ObservableBoolean(false);
+
+    public ObservableBoolean getIsEnabled() {
+        return isEnabled;
```

Comment:
what is enabled? I would prefer to change this name to more descriptive about this case e.g. isClearServiceEnabled. In the future, we can have more settings so it couldn't be obvious.